Core/Core/Dashboard/K5/View/Subject/K5SubjectHeaderView.swift

```diff
@@ -0,0 +1,60 @@
+//
```

Comment:
I would argue that K5 Subject files belong to the Dashboard Xcode group. I would move them to the Courses group under a K5 folder.

Reply:
agreed.